app/src/main/res/values/strings.xml

```diff
@@ -104,4 +104,6 @@
     <string name="selfie_instruction">to take a selfie to continue your registration.</string>
     <string name="recapture_tutorial">Click on the button to try capturing the tree again.</string>
     <string name="capture_tutorial">Click on the button to save tree.</string>
+    <string name="no_gps_device_header">NO GPS DEVICE FOUND</string>
```

Comment:
This can probably say  "No GPS found on device"

Reply:
same here, Swahili needed @sebastiangaertner